requirements/base.in

```diff
@@ -2,3 +2,4 @@
 
 django-object-actions==0.10.0           # Object actions in Django admin
 djangorestframework-xml==1.3.0
+django-multi-email-field==0.5.1         # Multi email input field in Django admin
```

Comment:
you'll need to run make upgrade to get this pulled into the other requirements files.